backend/src/main/kotlin/de/neuefische/backend/user/UserService.kt

```diff
@@ -0,0 +1,20 @@
+package de.neuefische.backend.user
+
+import org.springframework.stereotype.Service
+
+@Service
+class UserService(private val userRepo: UserRepo) {
+
+    fun getUserByName(name: String): UserProfileResponse? {
+        return userRepo.findByName(name)?.let { user ->
```

Comment:
Do you need to worry about case-sensitivity or not necessary? ( Maybe you validated this elsewhere, so ignore if not helpful!)
```suggestion
        return userRepo.findByName(name.lowercase())?.let { user -> 
```

-> also for mediums then if implemented:

`mediums = user.mediums?.map { it.lowercase } ?: emptyList()`

Reply:
for now I would like to keep it case-sensitive I think 🤔